### PR TITLE
CY-2601 React linter fixes

### PR DESCRIPTION
**Pull request by kubama**

### CODE REVIEW

File: widgets/events/src/EventsTable.jsx [hunk 1]
```diff
@@ -107,15 +108,15 @@ export default class EventsTable extends React.Component {
...
+                    fetchData={this.fetchGridData}
                     totalSize={data.total}
                     pageSize={widget.configuration.pageSize}
                     sortColumn={widget.configuration.sortColumn}
                     sortAscending={widget.configuration.sortAscending}
                     className="eventsTable"
                     noDataMessage={NO_DATA_MESSAGE}
                 >
-                    >
+                    &gt;
```
Comment (qooban) [inline widgets/events/src/EventsTable.jsx:119]: Hmmm... Do we actually need that?

**Review by qooban — COMMENTED** [review]
Huge work, very much appreciated!

Added few comments.

File: app/components/InstallWidgetModal.jsx [hunk 2]
```diff
@@ -34,6 +34,26 @@ export default class InstallWidgetModal extends Component {
         return !_.isEqual(nextState, this.state) || !_.isEqual(nextProps, this.props);
     }
 
+    openModal = () => {
```
[the remark this answers is not in the document]
Comment (kubama) [reply app/components/InstallWidgetModal.jsx:32]: Not necessary - removed

File: app/components/shared/graphs/PieGraph.jsx [hunk 3]
```diff
@@ -45,7 +45,7 @@ export default function PieGraph({ data }) {
             <PieChart>
                 <Pie dataKey="value" data={data} labelLine label isAnimationActive={false} cx="40%">
                     {data.map((entry, index) => (
-                        <Cell key={index} fill={entry.color} />
+                        <Cell key={entry.name} fill={entry.color} />
```
[the remark this answers is not in the document]
Comment (kubama) [reply app/components/shared/graphs/PieGraph.jsx:48]: Removed

File: widgets/blueprintCatalog/src/UploadModal.jsx [hunk 4]
```diff
@@ -0,0 +1,182 @@
+import ActionsPropType from './props/ActionsPropType';
+
+/**
+ * Created by pposel on 07/02/2017.
+ */
```
[the remark this answers is not in the document]
Comment (kubama) [reply widgets/blueprintCatalog/src/UploadModal.jsx]: Fixed

File: widgets/blueprints/src/BlueprintsTable.jsx [hunk 5]
```diff
@@ -0,0 +1,109 @@
+import BlueprintsViewPropTypes from './props/BlueprintsViewPropTypes';
+import BlueprintsViewDefaultProps from './props/BlueprintsViewDefaultProps';
+
+/**
+ * Created by kinneretzin on 08/01/2017.
+ */
```
[the remark this answers is not in the document]
Comment (kubama) [reply widgets/blueprints/src/BlueprintsTable.jsx]: Fixed

File: widgets/common/src/hooks/useBoolean.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+function useBoolean(initialValue = false) {
```
[the remark this answers is not in the document]
Comment (kubama) [reply widgets/common/src/hooks/useBoolean.js:7]: Addressed

File: widgets/events/src/EventsTable.jsx (hunk 1 above)
Comment (kubama) [reply widgets/events/src/EventsTable.jsx]: Most probably not - removed

File: widgets/common/src/hooks/useResetableState.js [hunk 7]
```diff
@@ -0,0 +1,9 @@
+function useResetableState(initialValue) {
```
[the remark this answers is not in the document]
Comment (kubama) [reply widgets/common/src/hooks/useResetableState.js]: You're right - renamed

**Review by qooban — APPROVED** [review]

**Pull request merged**